homeassistant/components/lcn/helpers.py

```diff
@@ -55,35 +67,37 @@
 }
 
 
-def get_device_connection(hass, address, config_entry):
+def get_device_connection(
+    hass: HomeAssistantType, address: AddressType, config_entry: ConfigEntry
+) -> DeviceConnectionType | None:
     """Return a lcn device_connection."""
     host_connection = hass.data[DOMAIN][config_entry.entry_id][CONNECTION]
     addr = pypck.lcn_addr.LcnAddr(*address)
     return host_connection.get_address_conn(addr)
 
 
-def get_resource(domain_name, domain_data):
+def get_resource(domain_name: str, domain_data: ConfigType) -> str:
     """Return the resource for the specified domain_data."""
     if domain_name in ["switch", "light"]:
-        return domain_data["output"]
+        return str(domain_data["output"])
```

Comment:
Cast instead of string copy:
```suggestion
        return Cast(str, domain_data["output"])
```